components/tools/OmeroWeb/omeroweb/feedback/views.py

```diff
@@ -180,7 +167,11 @@ def handler500(request):
     if request.is_ajax():
         return HttpResponseServerError(error500)
 
-    return custom_server_error(request, error500)
+    form = ErrorForm(initial={'error': error500})
```

Comment:
But doesn't it count as a breaking change, then? On this branch, that's probably not too serious, but I missed the fact on the `dev_5_1` branch.